restart_process/README.md

```diff
@@ -80,6 +80,13 @@ ERROR: Build Failed: ImageBuild: executor failed running [touch /tmp/.restart-pr
 ```
 this often means that your Dockerfile user ([see docs](https://docs.docker.com/engine/reference/builder/#user)) doesn't have permission to write to the file we use to signal a process restart. Use the `restart_file` parameter to specify a file that your Dockerfile user definitely has write access to.
 
+#### My args aren't being applied!
+Whatever you pass as the `entrypoint` parameter will override everything in your Dockerfile `COMMAND`, Dockerfile `ARGS`, and Kubernetes YAML `command`.
+
+If you still want to apply different arguments to different containers, you can specify `args` in your Kubernetes YAML. But _be warned_: these args only work if you pass your entrypoint as args (i.e. as a list of strings). So instead of `entrypoint="/my/great/bin"`*, pass `entrypoint=["/my/great/bin"]`.
```

Comment:
This feels like a bug and the docs feel more CYA than anything.

Would one of these work?
1. if entrypoint is a string and contains a space, return an error
2. if entrypoint is a string, `shlex.quote` it

I'd at least like to link to a GH issue here:
1. to make clear that we don't like the behavior, even if we don't have a solution worth implementing right now
2. to allow discussion/rejection of solutions such as above
3. to allow +1s

Reply:
1. but you're supposed to be able to pass an entrypoint like `cat myfile.txt > /bin/myapp`, so i don't think we could enforce this (unless at Tiltfile parse, we want to parse the k8s yaml and see if there are `args` specified and only complain then)
2. i'm not sure I understand how this addresses the problem

but yes it's definitely CYA 😅